smartapps/smartthings/keep-me-cozy-ii.src/keep-me-cozy-ii.groovy

```diff
@@ -36,7 +36,7 @@ preferences() {
 	section("Air conditioning setting...") {
 		input "coolingSetpoint", "decimal", title: "Degrees"
 	}
-	section("Optionally choose temperature sensor to use instead of the thermostat's... ") {
+	section("Optionally choose temperature sensor to use instead of the thermostat's...") {
```

Comment:
are there translations for these lines coming later?

Reply:
The translations are there but the space was messing the key